charts/karmada/_crds/bases/remedy/remedy.karmada.io_clusterstates.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
+  name: clusterstates.remedy.karmada.io
+spec:
+  group: remedy.karmada.io
```

Comment:
What does `remedy` mean?

Reply:
The remedy is used to represent the cure after a cluster failure in Karmada. I'll submit the proposal as soon as possible.